assets/dev/scss/direction/editor-dark-mode.scss

```diff
@@ -459,6 +459,26 @@ $devices: 'desktop', 'tablet', 'mobile';
 	.elementor-control-media__content {
 		border-color: $theme-dark-gray-light;
 	}
+
+	&.elementor-control-dynamic {
+
+		.elementor-control-gallery-clear {
+			border-color: $theme-dark-gray-light;
+		}
+	}
+
+	.elementor-control-dynamic-switcher {
+		border-bottom: 1px solid $theme-dark-gray-light;
+		border-left: 1px solid $theme-dark-gray-light;
+		border-radius: 0 3px 0 0;
+	}
+}
+
+body.rtl .elementor-control-type-gallery {
+	.elementor-control-dynamic-switcher {
+		border-right: 1px solid $theme-dark-gray-light;
+		border-left: none;
+	}
```

Comment:
This file shouldn't be changed since in version 3.0.0 this file will not exist anymore

Reply:
So how will these design changes be implemented?

Reply:
Let @joshmarom know about these changes

Reply:
Create a new WIP PR for this code